Include/internal/pycore_signal.h

```diff
@@ -64,7 +60,7 @@ struct _signals_runtime_state {
     } wakeup;
 
     /* Speed up sigcheck() when none tripped */
-    _Py_atomic_int is_tripped;
+    int is_tripped;
```

Comment:
Since the `_Py_atomic` types are not used anymore, can you add comments mentioning that these fields must be accessed using atomic ops?